cluster/ampagent/cmd/install.go

```diff
@@ -36,7 +36,7 @@ type InstallOptions struct {
 }
 
 var InstallOpts = &InstallOptions{}
-var Docker = ampdocker.NewClient(ampdocker.DefaultURL, ampdocker.DefaultVersion)
+var Docker *ampdocker.Docker
```

Comment:
Removing the initialization of the `Docker` global variable and then initializing it each time it's referenced is not great. It would be better to keep the initialization in a single place and avoid copy/pasting.

Reply:
right, I'll move the initialization back to the global variable assignment.